Sources/PackageDescription/Package.swift

```diff
@@ -185,6 +185,24 @@ public func ==(lhs: Package.Dependency, rhs: Package.Dependency) -> Bool {
 
 // MARK: Package Dumping
 
+struct Errors {
+    /// Storage to hold the errors.
+    private var errors = [String]()
+
+    /// Adds error to global error array which will be serialized and dumped in TOML at exit.
+    mutating func add(_ str: String) {
+        errors += ["\"\(str)\""]
```

Comment:
This will need to quote if we ever have an error with '"' in it... would be good to assert it doesn't for now.